### PR TITLE
insomnia: 7.1.1 -> core@2020.4.2

### PR DESCRIPTION
**Pull request by schonfinkel**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by babariviere — APPROVED** [review]
LGTM

**Review by SuperSandro2000 — COMMENTED** [review]
Why the unnecessary formatting changes? It makes the PR harder to review.

File: pkgs/development/web/insomnia/default.nix [hunk 1]
```diff
@@ -90,3 +129,4 @@ in stdenv.mkDerivation rec {
   };
 
 }
+
```
Comment (SuperSandro2000) [inline pkgs/development/web/insomnia/default.nix:132]:
```suggestion
```

File: pkgs/development/web/insomnia/default.nix [hunk 2]
```diff
@@ -14,14 +51,16 @@ let
     openssl
     stdenv.cc.cc
   ];
-in stdenv.mkDerivation rec {
+in
+stdenv.mkDerivation rec {
   pname = "insomnia";
-  version = "7.1.1";
+  version = "2020.4.2";
+
```
Comment (SuperSandro2000) [inline pkgs/development/web/insomnia/default.nix:58]:
```suggestion
```

File: pkgs/development/web/insomnia/default.nix [hunk 3]
```diff
@@ -14,14 +51,16 @@ let
     openssl
     stdenv.cc.cc
   ];
-in stdenv.mkDerivation rec {
+in
+stdenv.mkDerivation rec {
```
Comment (SuperSandro2000) [inline pkgs/development/web/insomnia/default.nix:55]:
```suggestion
in stdenv.mkDerivation rec {
```

File: pkgs/development/web/insomnia/default.nix [hunk 4]
```diff
@@ -1,9 +1,46 @@
-{ stdenv, makeWrapper, fetchurl, dpkg, alsaLib, atk, cairo, cups, dbus, expat
-, fontconfig, freetype, gdk-pixbuf, glib, gnome2, pango, nspr, nss, gtk3, gtk2
-, at-spi2-atk, gsettings-desktop-schemas, gobject-introspection, wrapGAppsHook
-, libX11, libXScrnSaver, libXcomposite, libXcursor, libXdamage, libXext
-, libXfixes, libXi, libXrandr, libXrender, libXtst, libxcb, nghttp2
-, libudev0-shim, glibc, curl, openssl, autoPatchelfHook }:
+{ stdenv
```
Comment (SuperSandro2000) [inline pkgs/development/web/insomnia/default.nix:1]: Any changes here? I don't feel like comparing 42 dependencies to find the one that changed.

Comment (mtrsk) [reply pkgs/development/web/insomnia/default.nix:1]: No changes, I've just used [nixpkgs-fmt](https://github.com/nix-community/nixpkgs-fmt).

**Comment by mtrsk** [comment]
> Why the unnecessary formatting changes? It makes the PR harder to review.

Sorry, the only changes for now it's in the version, the other divergences are due to running nixpkgs-fmt.

**Comment by SuperSandro2000** [comment]
> Sorry, the only changes for now it's in the version/sha/url, the other divergences are due to running nixpkgs-fmt.

I think they make it look worse especially with so many inputs. Can you please revert those parts?

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103251` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>insomnia</li>
  </ul>
</details>

**Comment by mtrsk** [comment]
> > Sorry, the only changes for now it's in the version/sha/url, the other divergences are due to running nixpkgs-fmt.
> 
> I think they make it look worse especially with so many inputs. Can you please revert those parts?

Sure, I should have done this earlier but forgot.

File: pkgs/development/web/insomnia/default.nix [hunk 5]
```diff
@@ -16,12 +16,11 @@ let
   ];
 in stdenv.mkDerivation rec {
   pname = "insomnia";
-  version = "7.1.1";
-
+  version = "2020.4.2";
```
Comment (SuperSandro2000) [inline pkgs/development/web/insomnia/default.nix:19]:
```suggestion
  version = "2020.4.2";

```
last nitpick

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103251` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>insomnia</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 103251` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>insomnia</li>
  </ul>
</details>

**Pull request merged**